spec/spec_helper.rb

```diff
@@ -40,4 +40,23 @@ module Types
   end
 
   config.include PredicatesIntegration
+
+  config.before do
```

Comment:
@solnic fyi, this is a copy & paste from dry-struct. We'll be able to remove container manipulations when https://github.com/dry-rb/dry-struct/issues/12 is resolved